src/Core/Geographic/Extent.js

```diff
@@ -296,14 +296,31 @@ Extent.prototype.offsetScale = function offsetScale(bbox) {
  * @param {type} bbox
  * @returns {Boolean}
  */
-Extent.prototype.intersect = function intersect(bbox) {
+Extent.prototype.intersects = function intersect(bbox) {
     const other = bbox.as(this.crs());
     return !(this.west() >= other.east(this._internalStorageUnit) ||
              this.east() <= other.west(this._internalStorageUnit) ||
              this.south() >= other.north(this._internalStorageUnit) ||
              this.north() <= other.south(this._internalStorageUnit));
 };
 
+/**
+ * @documentation: Return the intersection of this extent with another one
+ * @param {type} other
+ * @returns {Boolean}
+ */
+Extent.prototype.intersect = function intersect(other) {
```

Comment:
Is this the THREE.js nomenclature too? If not, I suggest to find another name to avoid the confusion with intersects. `getIntersection`?

Reply:
Yes it is:
`Box3.intersect` => ` Returns the intersection of this and box [...]`
`Box3.intersectsXXX` => `  Determines whether or not this box intersects XXX`

Reply:
So our `Extent.intersects` should be `Extent.intersectsExtent`? Pro: clearer, less likely to be mixed with `intersect`. Cons: a bit silly maybe :-)

Reply:
Ok, let's use `Extent.intersectsExtent`

Reply:
Done